packages/terra-graphs/docs/controls/Bar.md

```diff
@@ -237,6 +237,8 @@ Each bar can have a legendOptions object in [Values](#values) level.
 | x             | string   | Co-ordinate x, for plotting the bar |
 | y             | string   | Co-ordinate y, for height of bar    |
 
+**Note: providing invalid data to x or y will lead to console error.**
```

Comment:
nit:  not sure we need to specify "console" in the note.  I think just saying

_Note: providing invalid data to x or y will lead to an error._

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/24/commits/78be2902e1fe2cf435f3679ca146aaa79baa0cfd#diff-2884a9a901b097df95cad0c4d45b7f5b68b1ada145115e22a30ad0ee26811604R240).